doc/man/man3/notcurses_cell.3.md

```diff
@@ -56,9 +56,9 @@ typedef struct cell {
 
 **unsigned cell_styles(const cell* ***c***);**
 
-**void cell_styles_on(cell* ***c***, unsigned ***stylebits***);**
+**void cell_on_styles(cell* ***c***, unsigned ***stylebits***);**
```

Comment:
Ugh. I'd rather switch `cell_{on, off}_styles` in the actual API to match `ncplane_styles_*`, but Allah, the All-Powerful, has fucked us once again. Guess I just missed this in 2.0 API review. Bah! Thanks a lot for this find.

I'm going to go ahead and fix it in `USAGE.md` as well.

Reply:
Got it in `USAGE.md`. `cell_load_char()` was already correct there.